recipes-fsl/mcore-demos/imx-mcore-demos.inc

```diff
@@ -57,6 +57,6 @@ addtask deploy after do_install
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
 
-FILES:${PN} = "${nonarch_base_libdir}/firmware"
+FILES:${PN} = "${baselib}/firmware"
```

Comment:
It's `do_install` that needs to be fixed, installing to `nonarch_base_libdir` instead.

Reply:
Thank you @thochstein, fixed that